### DISCUSSION TITLE
Query logging

### DISCUSSION THREAD

**Discussion by lsfratel** [mako-framework/framework#310] [Q&A] [answered]
I've enabled query logging, but where do I see it?

**Comment by freost** [comment, marked as answer]
If you're working with a an app with backend rendered views then you can use the debug toolbar and inspect them there:

<img width="1665" alt="Screenshot 2023-01-24 at 13 05 00" src="https://user-images.githubusercontent.com/773958/214287455-9973d191-14dd-45f6-a47a-9ff680272eca.png">

If not then you can access the logs using `$this->database->getLogs()` or `$this->database->getConnection()->getLog()` for the logs of a specific connection.
